docs/install/redhat.rst

```diff
@@ -28,11 +28,27 @@ from the CrateDB package repository::
     # Import the public GPG key for verifying the package signatures.
     sudo rpm --import https://cdn.crate.io/downloads/yum/RPM-GPG-KEY-crate
 
-    # Register with the CrateDB package repository.
-    sudo rpm -Uvh https://cdn.crate.io/downloads/yum/7/x86_64/crate-release-7.0-1.x86_64.rpm
-
-The command above will install the ``/etc/yum.repos.d/crate.repo`` package
-repository configuration file.
+To register with the CrateDB package repository, create a file called ``cratedb.repo``
+in the ``/etc/yum.repos.d/`` directory for RedHat based distributions, or in the
+``/etc/zypp/repos.d/`` directory for OpenSuSE based distributions, containing::
+
+    [cratedb-stable]
+    name=CrateDB RPM package repository - $basearch - Stable
+    baseurl=https://cdn.crate.io/downloads/yum/7/$basearch
```

Comment:
ES doesn't use `$basearch`. Maybe we should also get rid of it?

Reply:
It works both with and without for me. Not sure which one is more standard-compliant, but the less the better?

Reply:
I am also for "the less the better" in general, yet quoting the documentation about `$basearch`, and it looks pretty standard amongst Red Hat at least, and may help if we ever release packages for ARM, for example?

> You can use `$basearch` to reference the base architecture of the system. For example, i686 machines have a base architecture of `i386`, and AMD64 and Intel 64 machines have a base architecture of `x86_64`.
>
> -- https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/deployment_guide/sec-using_yum_variables

Reply:
Yes I think as soon we release arm packages this is needed or? So lets keep it..